Test.cs

```diff
@@ -12,7 +14,12 @@ public static long calculate(long number)
                 throw new Exception("Number must be non-negative.");
             }
 
-            return number == 0 ? 1 : number * Calculate(number + 1);
```

Comment:
The method `GetBranchNama` contains a typo in the method name. It should be `GetBranchName`. Correcting typos ensures code readability and maintainability.